devito/operator/profiling.py

```diff
@@ -367,19 +368,14 @@ class AdvisorProfiler(AdvancedProfiler):
     _ext_calls = [_api_resume, _api_pause]
 
     def __init__(self, name):
-        self.path = locate_intel_advisor()
-        if self.path is None:
-            self.initialized = False
+        super().__init__(name)
+        path = get_advisor_path()
+        if path:
+            self._include_dirs = [path.joinpath('include').as_posix()]
+            libdir = path.joinpath('lib64').as_posix()
+            self._lib_dirs = [libdir]
```

Comment:
no, instances shouldn't change class attributes like that....

and btw why is it even a class attribute ?